controllers/argocdexport/reconcile.go

```diff
@@ -46,7 +45,7 @@ func setResourceWatches(bld *builder.Builder) *builder.Builder {
 	bld.For(&argoproj.ArgoCDExport{})
 
 	// Watch for changes to CronJob sub-resources owned by ArgoCDExport instances.
-	bld.Owns(&batchv1b1.CronJob{})
+	bld.Owns(&batchv1.CronJob{})
```

Comment:
There is already a PR [#631](https://github.com/argoproj-labs/argocd-operator/pull/631) merged for this. Can we rebase the PR with master?